docs/v2/getting-started.md

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
we hebben een Getting-started-v2.md en een v2/getting-started.md. hebben we niet maar 1 van de 2 nodig?